dataland-frontend/src/components/resources/frameworkDataSearch/DatasetDisplayStatusIndicator.vue

```diff
@@ -47,18 +42,19 @@ export default defineComponent({
     displayWarning(): boolean {
       return (
         this.displayedDataset?.currentlyActive === false ||
-        this.displayedDataset?.qaStatus === QAStatus.Pending ||
+        (this.displayedDataset?.qaStatus && this.displayedDataset.qaStatus !== QaStatus.Accepted) ||
         this.areMoreDatasetsViewableSimultaneously
       );
     },
     warningMessage(): string {
-      if (this.displayedDataset?.qaStatus === QAStatus.Pending) return "This dataset is currently pending review";
+      if (this.displayedDataset?.qaStatus === QaStatus.Pending) return "This dataset is currently pending review";
+      else if (this.displayedDataset?.qaStatus === QaStatus.Rejected) return "This dataset has been rejected";
       else if (this.displayedDataset?.currentlyActive === false) return "This dataset is superseded";
       else if (this.areMoreDatasetsViewableSimultaneously) return "You are only viewing a single available dataset";
       else return "ERROR";
     },
     buttonLabel(): string {
-      if (this.displayedDataset?.qaStatus === QAStatus.Pending || this.displayedDataset?.currentlyActive === false) {
+      if (this.displayedDataset?.qaStatus === QaStatus.Pending || this.displayedDataset?.currentlyActive === false) {
         return "View Active";
```

Comment:
If no active dataset exists but a dataset is pending, the EDIT DATA button doesn't do anything. Please change the behavior, s.t. the user is redirected to a page where the form is pre-filled with the content of the pending dataset.

Reply:
I disabled the edit button if the dataset is not accepted and active. The reason is that you should not build upon unaccepted datasets that possibly have wrong content. To be pushed.

Reply:
Sounds like a reasonable solution and it in the re-test i looks as described.